handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -93,6 +93,8 @@
         }
     }
 
+    final boolean startTls;
```

Comment:
I need it to be package-visible to access it in SniHandler to call the
static method to create an SslHandler from an SslEngine.
Another solution was to delegate the whole SslHandler (engine included) to
the context in SniHandler, but then we can't do the try/finally in case of
problem.
Or maybe we should do this try/finally in SslContext.newHandler in any case?

2016-08-31 14:40 GMT+02:00 Norman Maurer <notifications@github.com>:

> In handler/src/main/java/io/netty/handler/ssl/SslContext.java
> <https://github.com/netty/netty/pull/5178#discussion_r76979503>:
>
> > @@ -93,6 +93,8 @@
> >          }
> >      }
> >
> > +    final boolean startTls;
>
> private ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/netty/netty/pull/5178/files/f09c1d522ad12ab3ac88e77999f26345b3cc3eed#r76979503>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJ0zyHTlsApRITs7tz3OZGuRrPkkd0rks5qlXZTgaJpZM4IPuGt>
> .
>